src/main/java/de/tum/in/www1/artemis/web/rest/ComplaintResource.java

```diff
@@ -384,6 +384,41 @@ public ComplaintResource(AuthorizationCheckService authCheckService, ExerciseRep
         return ResponseEntity.ok(getComplaintsByComplaintType(complaints, complaintType));
     }
 
+    /**
+     * Get /courses/:courseId/exams/:examId/complaints/:complaintType
```

Comment:
This does not match the mapping.

Reply:
Ah yea, I forgot to remove the `:complaintType` part.